Makefile

```diff
@@ -77,3 +77,8 @@ install-kotlin-linters: ## install ktlint and detekt for linting Kotlin output
 	chmod a+x ktlint
 	test -f detekt-cli.jar || curl -sSL --output "detekt-cli.jar" https://github.com/detekt/detekt/releases/download/v1.23.6/detekt-cli-1.23.6-all.jar
 	echo "898dcf810e891f449e4e3f9f4a4e2dc75aecf8e1089df41a42a69adb2cbbcffa *detekt-cli.jar" | shasum -a256 -c -
+
+install-rust: ## install rustup and rust toolchain
+	curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+	echo 'source "$$HOME/.cargo/env"' >> ~/.bashrc  # Add Rust to bash startup
+	. ~/.bashrc  # Reload bashrc to include Rust in PATH
```

Comment:
I don't think we should have something that changes the user's environment as a make task. Let's do that in the CI config only. I can do that.

Reply:
Sounds good, let me know your preferred method for that, as I've implemented it in a lot of varied ways. Ideally, the CircleCI pre-built images are a good way to go, but it's a python application so that doesn't make sense in this case I imagine.

Reply:
https://github.com/mozilla/glean_parser/pull/773 landed, so you should be able to re-base on top of that and re-enable your tests.

Reply:
Awesome @badboy , thank you for landing that! Will do.